libcloudforensics/providers/aws/forensics.py

```diff
@@ -445,3 +446,36 @@ def CopyEBSSnapshotToS3(
   else:
     logger.info(
       'Image copy timeout. The process may be ongoing, or might have failed.')
+
+def InstanceNetworkQuarantine(zone: str,
+    instance_id: str,
+    exempted_src_subnets: Optional[List[str]] = None
+    ) -> None:
+  """Put an AWS EC2 instance in network quarantine.
+
+  Network quarantine is imposed via applying empty security groups to the
+  instance.
+
+  Args:
+    instance_id (str): : The id (i-xxxxxx) of the virtual machine.
+    exempted_src_subnets (List[str]): List of subnets that will be permitted
+  """
+  # Add /32 to any specified subnets that don't have a mask
+  # We're not checking the subnet is well formed, CreateIsolationSecurityGroup
+  # will take care of that
+  if exempted_src_subnets:
+    exempted_src_subnets[:] = [subnet if subnet.__contains__('/') else
+      subnet + '/32' for subnet in exempted_src_subnets]
+
+  try:
+    aws_account = account.AWSAccount(zone)
```

Comment:
This line can be moved out of the `try` block

Reply:
It can indeed, but I prefer the way it groups the code together from a style standpoint.